dash_bio/__init__.py

```diff
@@ -115,7 +110,6 @@
 _css_dist = []
 
 
-for _component in _components:
-    setattr(_this_module, _component.__name__, _component)
-    setattr(_component, '_js_dist', _js_dist)
-    setattr(_component, '_css_dist', _css_dist)
+for _component in __all__:
+    setattr(locals()[_component], "_js_dist", _js_dist)
+    setattr(locals()[_component], "_css_dist", _css_dist)
```

Comment:
not a big deal, but turns out since there's no separate CSS we can drop `_css_dist` entirely. [dcc](https://github.com/plotly/dash-core-components/blob/72875fc254dcfe461d77ae649effb51a0ecb16c9/dash_core_components_base/__init__.py#L142-L143) works this way for example.

Reply:
Dropped in 84ffb03   :scissors: